puppet/zulip/files/cron.d/delete-old-unclaimed-attachments

```diff
@@ -0,0 +1,7 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Weekly cronjob for cleaning up old, unclaimed attachments and their corresponding files.
+# Set up to run every Sunday at 7, an hour after our restart-server cronjob.
+0 7 * * 7 zulip /home/zulip/deployments/current/manage.py delete_old_unclaimed_attachments >/dev/null
```

Comment:
Should this be weekly like this or do we want daily or something like that?

Reply:
This has a bug -- we need to pass `-f` so it doesn't just do a dry run. 

I think daily makes sense.

Reply:
Ah, true, we need `-f`. Fixed that and changed to daily. Not sure if the hour matters?

Reply:
Likely not much.